spec/features/products_form_spec.rb

```diff
@@ -35,6 +35,24 @@
     end
   end
 
+  it "respects setting of label/value in Field::Select" do
+    old_release_year = ProductDashboard::ATTRIBUTE_TYPES[:release_year]
+
+    ProductDashboard::ATTRIBUTE_TYPES[:release_year] = Administrate::Field::Select.with_options(
+      collection: [
+        ["Last Year", 2019],
+        ["This Year", 2020]
+      ],
+    )
```

Comment:
this is a bit of a hacky way to test out the new functionality. Let me know if you'd prefer I add some arbitrary functionality to the example app to avoid the use of temporarily resetting this attribute type.

Reply:
["Last Year", 2019],